client/packages/portal-ui/src/lib/fullscreen-icon/FullscreenIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import { Icon } from '@equinor/eds-core-react';
+import { useEffect, useState } from 'react';
+
+export function FullscreenIcon() {
+  const [isFullscreen, setIsFullscreen] = useState(false);
+  useEffect(() => {
+    const eventSubscription = document.addEventListener(
+      'fullscreenchange',
+      () => {
+        setIsFullscreen(!!document.fullscreenElement);
+      }
+    );
+    return eventSubscription;
```

Comment:
Return type of document.addEventListener is void. This will not remove the event listener